code/game/objects/items/teleportation.dm

```diff
@@ -502,6 +511,8 @@
 		<b>Warning:</b> Teleporting into walls will activate a failsafe teleport parallel up to 3 meters, but the user will be ripped apart if it fails to find a safe location.<br>
 		<br>
 		Do not expose the teleporter to electromagnetic pulses. Unwanted malfunctions may occur.
+		<br>
+		Final word of caution: the technology involved is experimental in nature, you may feel lighter after heavy use. Both in weight, and in headedness. This is normal.
```

Comment:
```suggestion
		Final word of caution: the technology involved is experimental in nature. Although many years of research have allowed us to prevent leaving your organs behind, it simply cannot account for all of the liquid in your body.
```

The previous description feels really arcane and I'd rather we just spell it out.